### ISSUE TITLE
what preprocess need to do before send face image into mobile-id network

### ISSUE THREAD

**Issue by wangsheng1991** [liuziwei7/mobile-id#5]
for example,sub mean image or just divide some value

**Comment by xyt2008** [comment]
I met the same problem。can you tell us what preprocess need to do? @liuziwei7

**Comment by xialuxi** [comment]
I met the same problem。

**Comment by sunjunlishi** [comment]
width height

**Comment by sunjunlishi** [comment]
how aboult  result

**Comment by CLIsVeryOK** [comment]
the data in hdf format is stored in BGR not RGB, hdf data is single format and has a relation with uint8 RGB data as : RGB = (SingleHdf + 1.6) * 80.  I am not sure, but for me, it works.

**Comment by liuziwei7** [comment]
Yes, the data in HDF5 file is stored in the BGR order channel-wise for accommodating with the Caffe convention. The value of the images has been normalized as: data = data * (3.2/255) - 1.6 .

Also, the script for crop&align face images ("align_face_celeba.m") has been uploaded to the repository.
